ratpack-bearer-auth/src/main/java/st/ratpack/auth/springsec/SpringSecCheckAuthModule.java

```diff
@@ -3,8 +3,10 @@
 import com.google.inject.*;
 import com.google.inject.multibindings.OptionalBinder;
 import ratpack.guice.ConfigurableModule;
-import ratpack.http.client.HttpClient;
 import st.ratpack.auth.*;
+import st.ratpack.auth.springsec.jwt.DefaultJWKProvider;
+import st.ratpack.auth.springsec.jwt.JWKProvider;
+import st.ratpack.auth.springsec.jwt.JWTTokenValidator;
```

Comment:
JWT isn't very common with SpringSecurity we probably don't want to have this in the spring security module.